styles.css

```diff
@@ -132,11 +126,13 @@
 .unique {
   display: flex;
   justify-content: center;
+  color: white;
```

Comment:
You should use the color #e5e5e5 for this section as stated in the design/requirement. with this change, the user will be able to notice that he is actually about to start filling out a survey.

Reply:
Greate job so far now fulfills the UI design request.